assets/email_template.html

```diff
@@ -43,13 +43,6 @@ <h3>Pipeline Configuration:</h3>
     </tbody>
 </table>
 
-<h3>Software Versions:</h3>
```

Comment:
I guess I'll find the answer to this later in the review.. but how come you removed this?